raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/grammar/BaseRamlGrammar.java

```diff
@@ -759,6 +759,6 @@ protected Rule protocols()
 
     protected Rule responseCodes()
     {
-        return range(Range.closed(100L, 599L));
+        return range(Range.closed(0L, 999L));
```

Comment:
open a spec bug to clarify what should be the allowed range of status codes, otherwise next time somebody will want to use status code 1050 and it will not be possible.